WireAuthentication/Sources/WireAuthenticationUI/Views/DetermineAuthMethod/DetermineAuthMethodViewModel.swift

```diff
@@ -251,9 +253,8 @@ package final class DetermineAuthMethodViewModel: ObservableObject {
                 // No op
                 break
             case .noDefaultCodeAvailable:
-                router.presentSheet(
-                    RootView.ModalDestination.authFlow(backendInfo: backendInfo)
-                )
+                router.popToRoot() // in case backend changed, we force SwiftUI to re-render the navigation
```

Comment:
@johnxnguyen here's the fix

Reply:
Another solution could be to push `determineAuth` again, this way the user could go backwards. But I think also find as it is.

suggestion: comment could be something like: "clear the navigation stack before replacing root"

Reply:
good point, i'll update the comment, and keep the solution for now